README.md

```diff
@@ -33,6 +33,29 @@ $ docker --help
 
 This should display the list of commands and features.  If it fails with unknown command the installation did not work correctly.
 
+###Several facts which good to know about work with Docker in Linux
```

Comment:
For correct rendering, MD needs a space after the heading chars: `### Several`.

 This might be better as: `Several useful facts for working with Docker in Linux`